src/geofr/management/commands/populate_drainage_basins.py

```diff
@@ -14,7 +14,7 @@
     '03': 'Seine-Normandie',
     '04': 'Loire-Bretagne',
     '05': 'Adour-Garonne',
-    '06': 'Rhône- Méditérranée',
+    '06': 'Rhône-Méditérranée',
```

Comment:
```suggestion
    '06': 'Rhône-Méditerranée',
```

C'est « Rhône-Méditerranée » pas « Rhône-Méditérranée ».